app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<%= @user.id %>
+<%= form_for @post do |form| %>
+  <%= form.label :message %>
+  <%= form.text_area :message %>
+<%= form.hidden_field :wall_id, :value => @user.id %>
+  <%= form.submit "Submit" %>
+<% end %>
+
+
+
+ <h1>  users own  wall </h1>
```

Comment:
... could we maybe put the User's name here e.g. _Katy's own wall_ or _Katy's wall_; in-lieu of 'users own wall'?

Reply:
Added. New commit sent.